src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/SqlBulkCopyTest.cs

```diff
@@ -247,5 +247,43 @@ public void DestinationTableNameWithSpecialCharTest()
         {
             DestinationTableNameWithSpecialChar.Test(srcConstr, AddGuid("SqlBulkCopyTest_DestinationTableNameWithSpecialChar"));
         }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureServer))]
```

Comment:
If you use the **Northwind** database instead of **tempdb**, you can run your test for the **Azure** database too.